kivy/core/window/window_sdl2.py

```diff
@@ -392,6 +392,12 @@ def _get_window_pos(self):
     def _set_window_pos(self, x, y):
         self._win.set_window_pos(x, y)
 
+    def set_window_opacity(self, opacity):
+        return self._win.set_window_opacity(opacity)
```

Comment:
Again unnecessary `return`. Also see how the rest of such functions are named and used in the file.